datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/InstantDeserializer.java

```diff
@@ -92,7 +95,8 @@ public class InstantDeserializer<T extends Temporal>
             a -> ZonedDateTime.ofInstant(Instant.ofEpochSecond(a.integer, a.fraction), a.zoneId),
             ZonedDateTime::withZoneSameInstant,
             false, // keep zero offset and Z separate since zones explicitly supported
-            DEFAULT_NORMALIZE_ZONE_ID 
+            DEFAULT_NORMALIZE_ZONE_ID,
+            DEFAULT_READ_NUMERIC_STRINGS_AS_DATE_TIMESTAMP
```

Comment:
Should this match the feature toggle name?

Reply:
Yes, good point it should.